isobus/include/isobus/isobus/can_message.hpp

```diff
@@ -11,6 +11,7 @@
 #ifndef CAN_MESSAGE_HPP
 #define CAN_MESSAGE_HPP
 
+#include "isobus/isobus/can_constants.hpp"
```

Comment:
In the removed `isobus/isobus/can_address_claim_state_machine.hpp` file there was this include `isobus/isobus/can_constants.hpp`. Since the address claim state machine header was included in the internal control function header, which in turn was included in the network manager. In short meaning; when one file previously included the `isobus/isobus/can_network_manager.hpp` file they also got the constants file. Since this link got removed a ton of of files were missing this header, so I assumed it was reasonable to put it here instead of in every file that was missing it... What do you think?

Reply:
I guess I generally prefer to minimize the scope of includes, that's OK with me